pkg/pods/testdata/taskrunspec-not-yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 The Tekton Authors.
```

Comment:
Note that if it doesn't have the `.yaml` extension, it can still be a valid yaml file. But I think this is already something used in other part of `tkn` as well, so it's fine as it.